west.yml

```diff
@@ -88,7 +88,7 @@ manifest:
       revision: 31acbaa36e9e74ab88ac81e3d21e7f1d00a71136
       path: modules/lib/gui/lvgl
     - name: mbedtls
-      revision: 24d84ecff195fb15c889d9046e44e4804d626c67
+      revision: pull/20/head
```

Comment:
Really? Do you need this one downstream?

Reply:
well, it does include a header change and as it is part of a commit with changes that also updates the manifest I don't see why this should be a problem.
As long as the upstream PR is merge, the SHA will be in the repo, and there will not be a need for an NCS fork.

Reply:
So I guess you'll update this to `fromlist` and drop the DNM when your upstream PR is merged, right?

Reply:
yep. as soon as https://github.com/zephyrproject-rtos/zephyr/pull/34594 is merged, things will be `fromtree`.
but that would still require the mbedtls pr to be merged.